src/runtimes/node/bundlers/esbuild/bundler.ts

```diff
@@ -147,6 +155,31 @@ let require=___nfyCreateRequire(import.meta.url);
       srcFile,
       outputExtension,
     })
+
+    for (const [filename] of Object.entries(metafile.outputs)) {
```

Comment:
I think `metafile.outputs` will contain all the files written by esbuild, which can include a source map. It would be worth checking the extension and limit this operation to JavaScript files.

Reply:
Good idea, done in https://github.com/netlify/zip-it-and-ship-it/pull/1532/commits/695f62241865f52faae5108e9fd6311c9cb1ccfd.